uninstall/uninstall.sh

```diff
@@ -6,4 +6,196 @@ APP_PATH=`cd "$APP_PATH"; pwd`
 HOME_PATH=~
 INSTALL_PATH="${HOME_PATH}/.splashkit"
 
-rm -rf ${INSTALL_PATH}
+SKM_PATH=`cd "$APP_PATH/.."; pwd`
+
+HAS_PYTHON3=false
+
+source "${SKM_PATH}/tools/set_sk_env_vars.sh"
```

Comment:
Does this work to set win64 in the arm version?

Reply:
win64 works, but...

As mentioned above - Will need to look into improved testing with this as unable to get any errors in my current Windows ARM VM when just using the regular MINGW64 terminal.